### ISSUE TITLE
Blaze: Manage Campaigns - Show Blaze overlay only once

### ISSUE THREAD

**Issue by momo-ozawa** [wordpress-mobile/WordPress-Android#18787]
Ref: p1689601960878889-slack-C04LJFS1G5P
```[tasklist]
### Tasks
- [x] Show Blaze overlay only once (not once per site or user, just once overall) #18843
```

**Comment by zwarm** [comment]
The "Learn more" feature was a late addition and is out of scope for Android. It will be taken up during the Android dashboard personalization phase of the UI Modernization project. See issue -  https://github.com/wordpress-mobile/WordPress-Android/issues/18882
